modules/core/04-channel/v2/keeper/packet.go

```diff
@@ -54,21 +54,28 @@ func (k *Keeper) sendPacket(
 		return 0, "", errorsmod.Wrapf(types.ErrInvalidPacket, "constructed packet failed basic validation: %v", err)
 	}
 
+	// Before we do client keeper level checks, we first get underlying base clientID
+	clientId := sourceClient
+	baseClientId, ok := k.GetBaseClient(ctx, sourceClient)
+	if ok {
+		clientId = baseClientId
+	}
```

Comment:
Then this could be something like:
```suggestion
	if resolvedClientId, hasAlias := k.GetBaseClient(ctx, sourceClient); hasAlias {
		clientId = resolvedClientId
	}
```